01-introduction.Rmd

```diff
@@ -2,68 +2,65 @@
 
-Even if we did have perfect clarity on the topic, there's still the issue of helping education systems learn to leverage these new analytical tools. In many education settings, school administrators and their staff may have never had someone around who deeply understands education, knows how to write code, and uses statistical techniques all at once, as data science in education could be defined [@conway2010data].
+Even then, there's still the issue of how education systems can best use data science tools. In many education settings, school staff have never worked with someone with this combination of skills -- education content knowledge, programming, and statistics [@conway2010data].
 
-## Making the path a little clearer
+## Making the path clearer
 
-As data science in education grows, the way we talk about and conceptualize it also needs to grow; doing so can help us advance data science in education as a discipline and speak to the unique opportunities and concerns that arise with analyzing data in our domain.
+As data science in education grows, the way we think about it also needs to grow. Doing so can help us advance data science in education as a discipline. It will uncover the unique opportunities that come with analyzing data in our domain.
 
-We begin this book by offering a primer for data science in education, including a discussion of unique challenges and foundational skills in the programming language R. This includes this chapter as well as suggestions for how to use this text ([Chapter 2](#c02)), our definition of the process of data science and what it "looks like" in terms of who does data science and how they do it ([Chapter 3](#c03)), and a discussion of data science in education in the context of the wider fields of both education and data science ([Chapter 4](#c04)).
+With this chapter, we begin by offering a primer for data science in education, including a discussion of foundational skills in the programming language R. The primer continues with suggestions for how to use this text ([Chapter 2](#c02)), our definition of the data science process, what it looks like in different roles ([Chapter 3](#c03)), and a discussion of similarities and differences between data science in education and data science in others fields ([Chapter 4](#c04)).
 
-Next, you'll take what you've learned and apply it in our data analysis in education walkthroughs. The walkthroughs in this book are our contribution towards a more example-driven approach to learning. They're meant to make the ambiguous path of learning data science in education a little clearer by way of recognizable and actionable demonstrations. 
+Next, you'll take what you've learned and apply it in education data analysis walkthroughs. The walkthroughs are meant to be an example-driven approach to learning. They include recognizable and actionable demonstrations. 
```

Comment:
```suggestion
Next, you'll take what you've learned and apply it in data analysis walkthroughs using education data. The walkthroughs are meant to aid learning through practical examples. They include recognizable and actionable demonstrations. 
```

Reply:
Should we call these "walkthroughs" or "data science walkthroughs"? Does data analysis = data science?